crates/codegen/runtime/cargo/src/runtime/napi_interface/cst.rs

```diff
@@ -84,7 +84,9 @@ impl NonTerminalNode {
     }
 
     #[napi(catch_unwind, js_name = "toJSON")]
-    /// Serialize the token node to JSON.
+    /// Serialize the node to JSON.
+    ///
+    /// This method is intended for debugging purposes and may not be stable.
```

Comment:
We agreed to remove this IIUC, so I think we should rather drop this from the other function (my mistake) rather than add it here

Reply:
Sounds good.